test/runner/provider/test.js

```diff
@@ -100,6 +101,33 @@ define([
             .init();
     });
 
+    QUnit.test('extend default API with APIP data', function(assert) {
+        var ready = assert.async();
+        assert.expect(2);
+
+        itemRunner.register('qti', qtiRuntimeProvider);
+        itemRunner('qti', itemData)
+            .on('init', function() {
+                assert.ok(typeof this.getApipData === "function", 'The item getApipData method presented');
+                assert.ok( _.isNull(this.getApipData()), 'item runner getApipData returns correct value for an absent data');
+                ready();
+            })
+            .init();
+    });
+
+    QUnit.test('reading APIP data from QTI item runner', function(assert) {
+        var ready = assert.async();
+        assert.expect(1);
+
+        itemRunner.register('qti', qtiRuntimeProvider);
+        itemRunner('qti', itemApipData)
+            .on('init', function() {
+                assert.ok( _.isPlainObject(this.getApipData()), 'item runner getApipData returns correct value');
```

Comment:
The test is not complete. Nothing prevents it to load any data but the expected one.
You should rather perform a `deepEqual` with an expected data set.